pyperformance/venv.py

```diff
@@ -24,6 +24,10 @@ def from_benchmarks(cls, benchmarks):
         for bench in benchmarks or ():
             filename = bench.requirements_lockfile
             self._add_from_file(filename)
+            if bench.setup_py:
+                # pip doesn't support installing a setup.py,
+                # but it does support installing from the directory it is in.
+                self._add(os.path.dirname(bench.setup_py))
```

Comment:
It might be better to handle this in `_pip.install_requirements()`, since it is a pip quirk, but I'm not going to stress over it.

```suggestion
                self._add(bench.setup_py)
```

(See the following comment for the `_pip.install_requirements()` changes.)

Again, I'm not going to block merging on this, though you're welcome to take my suggestion. 🙂